GameOfRunes/GameOfRunes/Models/EnemyWaveRealmModel.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  EnemyWaveRealmModel.swift
+//  GameOfRunes
+//
+//  Created by Jermy on 31/3/20.
+//  Copyright © 2020 TeamHoWan. All rights reserved.
+//
+
+import RealmSwift
+
+class EnemyWaveRealmModel: Object {
```

Comment:
Right now the Levels are represented as EnemySpawnUnit, which can be concatenated together to form larger and larger units that then form a level. Did you implement this EnemyWaveRealModel type because EnemySpawnUnit cannot be added to Realm easily?

Reply:
Realm can't have nested arrays/list. So this class is used as a wrapper.